app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1745,7 +1746,9 @@ class BrowserTabFragment :
             }
             is Command.SwitchToTab -> {
                 binding.focusedView.gone()
-                viewModel.autoCompleteSuggestionsGone()
+                if (binding.autoCompleteSuggestionsList.isVisible) {
```

Comment:
Added this condition everywhere to ensure we don't send any pixels (this happens in `autoCompleteSuggestionsGone()`) if the `autoCompleteSuggestionsList` is hidden more than once (noticed some edge-cases).